packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -77,7 +92,10 @@ class CupertinoPageScaffold extends StatelessWidget {
               top: topPadding,
             ),
           ),
-          child: child,
+          child: Padding(
```

Comment:
Sorry, the linter doesn't currently pop a warning for these but we're still using Dart1 syntax for the time being. `new Padding` here.